components/src/dynamo/sglang/tests/test_sglang_decode_handler.py

```
@@ -262,6 +265,33 @@ def test_build_sampling_params_forwards_repetition_controls_for_openai_requests(
 
 
+class TestMultimodalGuard:
+    """Tests for multimodal guard when frontend extraction is missing."""
+
+    IMAGE_MESSAGE = {
+        "role": "user",
+        "content": [
+            {"type": "image_url", "image_url": {"url": "https://example.com/a.jpg"}},
+            {"type": "text", "text": "describe image"},
+        ],
+    }
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)